module/VuFind/src/VuFind/Controller/CommentsController.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * Comments Controller
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Controller
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFind\Controller;
+
+use VuFind\Exception\Forbidden as ForbiddenException;
+use VuFind\Validator\CsrfInterface;
+
+/**
+ * Comments controller.
+ *
+ * @category VuFind
+ * @package  Controller
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class CommentsController extends AbstractBase
+{
+    /**
+     * Get all comments for the logged in user
+     *
+     * @return View
+     */
+    public function userCommentsAction()
```

Comment:
I wonder if there's a better, less redundant action name -- one that would result in a cleaner URL and require less complicated logic for building the route name for redirection. Maybe something like userListAction or userOwnedAction or userCreatedAction?